src/create-accounts/create-accounts.service.ts

```diff
@@ -323,32 +331,40 @@ export class CreateAccountsService {
       college_id
     );
 
-    await this.redis.set(`auditor_password:${user.id}`, user.password, 'EX', 2 * 24 * 60 * 60);
+    await this.redis.set(
+      `auditor_password:${user.id}`,
+      user.password,
+      'EX',
+      2 * 24 * 60 * 60
+    );
 
-    setTimeout(async () => {
-      await this.userRepository.update(user.id, { password: null });
-    }, 2 * 24 * 60 * 60 * 1000); 
+    setTimeout(
+      async () => {
+        await this.userRepository.update(user.id, { password: null });
+      },
+      2 * 24 * 60 * 60 * 1000
+    );
 
-  
     const loginUrl = `${process.env.FRONTEND_URL}/auth/login}`;
```

Comment:
The frontend url is https://frontend-mvp-gt99.vercel.app/signIn. Why is yours different?

Reply:
I will change it